### PR TITLE
Assist on Android: add shortcut, access from lock screen

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Assist on Android: 
- add procedure on adding Assist shortcuts
- add step on starting Assist from the lock screen

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by jpelgrom** [comment]
This shortcut procedure might work but it's not for Assist. For Assist shortcuts it is much easier: just open the widgets panel on your home screen and drag Assist from it.

**Comment by c0ffeeca7** [comment]
> This shortcut procedure might work but it's not for Assist. For Assist shortcuts it is much easier: just open the widgets panel on your home screen and drag Assist from it.

:laughing: so easy! I did not expect this.

File: source/voice_control/android.markdown [hunk 1]
```diff
@@ -47,6 +49,14 @@ Once Assist has been defined as the default digital assistant on your phone, you
...
    - Speak your command.
 
+### Using Assist via shortcut
+
+1. On your phone, open the **Widgets** panel.
+2. From the list, select **Home Assistant**.
+3. Long tap the **Assist** icon and drag it onto your home screen.
+4. Select the server and the assistant.
+5. If you want to be able to use voice control, enable the **Start listening** toggle.
+6. Repeat this procedure for each server you want to connect to, for example if you support someone elses home.
```
Comment (frenck) [inline source/voice_control/android.markdown]:
```suggestion
6. Repeat this procedure for each server you want to connect to, for example, if you support someone else's home.
```

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**